scripts/run-mod-settings.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+
+set -e
+U=http://localhost:9130
+T=testlib15
+username=testing_admin
+password=admin
+curl -d"{\"id\":\"$T\"}" $U/_/proxy/tenants
+curl -d'{"name":"DB_HOST","value":"localhost"}' $U/_/env
+curl -d'{"name":"DB_PORT","value":"5432"}' $U/_/env
+curl -d'{"name":"DB_USERNAME","value":"postgres"}' $U/_/env
```

Comment:
In the top-level `README.md`, we are told to go with usr=folio, password=folio, database=folio_modules. (That's what I'm using.) The script should probably be aligned with this.